pom.xml

```diff
@@ -25,7 +25,7 @@
 	<name>Maven Integration for Eclipse (root)</name>
 
 	<properties>
-		<rcptt-maven-version>2.5.4</rcptt-maven-version>
+		<rcptt-maven-version>2.5.5-MO</rcptt-maven-version>
```

Comment:
There is already an M3 available.
```suggestion
		<rcptt-maven-version>2.5.5-M3</rcptt-maven-version>
```

Reply:
Good point. Perhaps I shouldn't simply pick the first entry 😅 

![image](https://github.com/eclipse-m2e/m2e-core/assets/70652978/34f3b97d-1970-46e5-9913-994015bdeb2f)

Reply:
`M3` isn't even listed there :O, but the repo from the pom contains it:
https://repo.eclipse.org/content/repositories/rcptt-releases

Reply:
The tests passes with the M2. Should I add the Eclipse repo and switch to the M3 or keep the one from Maven Central?

Reply:
Awesome. 🎉
The repo is already configured in the rcptt project pom.xml.
So we can just go to M3. AFAICT the rcptt maven plugin is not at maven-central.
But since that property is only used in one place, could you please take this occasion and just inline it into the m2e.rcptt.tests plugin?

Reply:
Done